ui/imports/shared/controls/chat/UserImage.qml

```diff
@@ -31,6 +31,14 @@ Loader {
                 border.width: 1
                 border.color: Style.current.border
                 source: {
+                    if (root.isReplyImage) {
+                        if (root.profileImage) {
+                            return root.profileImage
+                        }
+                        if (root.identiconImageSource) {
+                            return root.identiconImageSource
+                        }
+                    }
                     if (profileImageSource) {
```

Comment:
I know it's not your changes, but i can't find this property or object in this file. Its can be a problem?

Reply:
It's a "global" somewhere higher. I think it is getting refactored soon

Reply:
okay :)